app/src/main/java/com/jetpack/compose/learning/magnifierview/MagnifierViewActivity.kt

```diff
@@ -0,0 +1,137 @@
+package com.jetpack.compose.learning.magnifierview
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.view.MotionEvent
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.*
```

Comment:
Please remove wildcard imports and use full qualified imports as per general practice. You can take a reference from ZoomViewActivity.kt class.